pkgs/shells/zsh/zsh-git-prompt/default.nix

```diff
@@ -14,7 +14,7 @@
 #
 # Or if installed locally:
 #
-#   source ~/.nix-profile/share/zsh-git-prompt/zshrc.sh
+#   source $XDG_DATA_HOME/nix/profile/share/zsh-git-prompt/zshrc.sh
```

Comment:
```suggestion
#   source $XDG_STATE_HOME/nix/profile/share/zsh-git-prompt/zshrc.sh
```

Reply:
This won't work when XDG_STATE_HOME is not set which it isn't by default IIRC